MotorI2C/motorI2C.cpp

```diff
@@ -0,0 +1,57 @@
+#include <Wire.h>
+#include "MotorI2C.h"
+#include "SmartMotor.h"
+
+// Create a custom I2C instance for the Pico (as slave)
+TwoWire I2C_Slave = TwoWire(0);  // Use bus 0 for the I2C
+
+MotorI2C::MotorI2C(TwoWire& wire, int slaveAddress) 
+  : wireInstance(wire), slaveAddress(slaveAddress), receivedLeftSpeed(0), receivedRightSpeed(0), newDataAvailable(false), lastCommandTime(0) {}
+
+// Setup I2C communication
+void MotorI2C::begin() {
+  wireInstance.begin(slaveAddress);  // Initialize the I2C as a slave
```

Comment:
The wire is already started in the `setup()`. This is the error Marco was talking about